src/main/java/seedu/address/model/Model.java

```diff
@@ -123,6 +125,9 @@
      */
     void setPerson(Person target, Person editedPerson);
 
+    /** Returns a list of persons at the provided indexes */
+    ObservableList<Person> getPersonsByIndexes(Set<Index> indexes);
```

Comment:
Eh... I'd recommend implementing getPersonByIndex instead for more overall usability, but this is fine.